code/model/SiteTree.php

```diff
@@ -2643,19 +2609,18 @@ public function setMenuTitle($value) {
 	}
 	
 	/**
-	 * A flag provides the user with additional data about the current page status,
-	 * for example a "removed from draft" status. Each page can have more than one status flag.
-	 * Returns a map of a unique key to a (localized) title for the flag.
-	 * The unique key can be reused as a CSS class.
-	 * Use the 'updateStatusFlags' extension point to customize the flags.
+	 * A flag provides the user with additional data about the current page status, for example a "removed from draft"
+	 * status. Each page can have more than one status flag. Returns a map of a unique key to a (localized) title for
+	 * the flag. The unique key can be reused as a CSS class. Use the 'updateStatusFlags' extension point to customize
+	 * the flags.
 	 * 
 	 * Example (simple): 
-	 * "deletedonlive" => "Deleted"
+	 *   "deletedonlive" => "Deleted"
 	 * 
 	 * Example (with optional title attribute): 
-	 * "deletedonlive" => array('text' => "Deleted", 'title' => 'This page has been deleted')
+	 *   "deletedonlive" => array('text' => "Deleted", 'title' => 'This page has been deleted')
 	 *
-	 * @param Boolean $cached
+	 * @param bool $cached Whether to serve the fields from cache; false regenerate them
```

Comment:
The parameter has a default value which means null is a valid call value for that parameter:

```php
getStatusFlags() is the same as getStatusFlags(null)
```

"Why bother?" I hear you thinking. Because I can hear your thoughts. PHP supports parameters [with default values] between others that don't have default values:

```php
function getSomeValue($type, $cached = true, $shouldReturn)...
```

The `null` type hint tells the type checker that `getSomeValue("field", null, true)` is a valid call.